java-components/adhoc-analyser-tool/src/main/java/com/redhat/hacbs/analyser/kube/ResetDependencyBuildsCommand.java

```diff
@@ -36,7 +36,7 @@ public void run() {
             } else {
                 List<DependencyBuild> items = client.list().getItems();
                 for (var request : items) {
-                    if (!failedOnly || request.getStatus().getState().equals("DependencyBuildFailed")) {
+                    if (!failedOnly || request.getStatus().getState().equals("DependencyBuildStartFailed")) {
```

Comment:
missed this in previously PR etc. examinations

why do we have java runnables manipulating status for a CRD being handled by our golang operartor / controller?
should be be concerned with multiple writers getting in the way of each other, possibly corrupting the flow through the state machine?

Reply:
Just to resolve the conversation here, this is a testing utility that makes it easy to retry failed jobs when doing local development. It is not actually running on the cluster.

Reply:
yep per our conversation last night, occasionally my old eyes struggle with my 4K monitor :-)